.github/workflows/integration-tests-github-tier-0.yml

```diff
@@ -0,0 +1,71 @@
+name: GitHub integration tests tier 0
+
+on:
+  pull_request:
+    paths:
+        - ".github/workflows/integration-tests-github-tier-0.yml"
+        - "src/modulesd/**"
+        - "src/Makefile"
+        - "tests/integration/conftest.py"
+        - "tests/integration/test_github/**"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version-file: ".github/workflows/.python-version-it"
+          architecture: x64
+      # Build wazuh server for linux.
+      - name: Build wazuh server for linux
+        run: |
+          make deps -C src TARGET=server -j2
```

Comment:
I think we should compile for agent Linux and Windows. @QU3B1M is working on this, talk to him.